river/Cursor.zig

```diff
@@ -58,9 +160,11 @@ const Mode = union(enum) {
         view: *View,
         delta_x: f64 = 0,
         delta_y: f64 = 0,
-        /// Offset from the lower right corner of the view
-        offset_x: i32,
-        offset_y: i32,
+        resize_direction: ResizeDirection,
```

Comment:
Since this is usually accessed as `resize.direction` or similar, what do you thing about dropping the usually redundant prefix?
```suggestion
        direction: ResizeDirection,
```